contributing/code-style-and-quality-preamble.md

```diff
@@ -0,0 +1,27 @@
+---
+title:  "Apache Flink Code Style and Quality Guide — Preamble"
+---
+
+{% include code-style-navbar.md %}
+
+
+_Preamble_
```

Comment:
I think to remove this, it already says Preamble on top of the nav bar

<img width="835" alt="Screen Shot 2019-07-08 at 1 12 25 PM" src="https://user-images.githubusercontent.com/1891970/60832691-216b2080-a182-11e9-870a-ce1a4b0a4592.png">